Formula/mesos.rb

```diff
@@ -66,32 +66,6 @@ def install
     (lib/"python2.7/site-packages").mkpath
     (lib/"python2.7/site-packages/homebrew-mesos-boto.pth").write "#{boto_path}\n"
 
-    # work around distutils abusing CC instead of using CXX
-    # https://issues.apache.org/jira/browse/MESOS-799
-    # https://github.com/Homebrew/homebrew/pull/37087
-    native_patch = <<-EOS.undent
-      import os
-      os.environ["CC"] = os.environ["CXX"]
-      os.environ["LDFLAGS"] = "@LIBS@"
-      \\0
-    EOS
-    inreplace "src/python/native/setup.py.in",
-              "import ext_modules",
-              native_patch
```

Comment:
I'll try with that put back in. Any idea why upstream seems to think it is resolved? https://issues.apache.org/jira/browse/MESOS-799